README.md

```diff
@@ -20,31 +20,76 @@ Emacs users may install the package from MELPA, and then add the following to th
 
 ## Spacemacs
 
-Add
-```lisp
-(vue-mode :location (recipe
-                     :fetcher github
-                     :repo "codefalling/vue-mode"))
-```
-
-to your `packages`.and
-
-```lisp
-(defun your-layer-name/init-vue-mode ()
-  (use-package vue-mode))
-```
-
-to your `package.el`.
-
-And you could custom the region background color if you need (default is highlight):
-
-```lisp
-(defun your-layer-name/init-vue-mode ()
-  (use-package vue-mode
-    :config
-    ;; 0, 1, or 2, representing (respectively) none, low, and high coloring
-    (setq mmm-submode-decoration-level 0)))
-```
+### There are [two ways](http://spacemacs.org/doc/DOCUMENTATION.html#configure-packages) to install and configure Spacemacs Packages:
+
+#### A. The fastest way: Adding the necessary configurations to the default configuration file `.spacemacs`
+
+   1. Within any screen in Spacemacs (including the Splash Screen, after loading), press `SPC` `f` `e` `d`.
+   2. This will bring you to the edit screen of the `.spacemacs` file.
+   3. Locate the section named `dotspacemacs-additional-packages '()`
+   4. The following code should go inside the above parenthesis:
+   ```lisp
+   (vue-mode :location (recipe
+                        :fetcher github
+                        :repo "codefalling/vue-mode"))
+   ```
+   • So the section should look like this, after the update:
+   ```lisp
+   dotspacemacs-additional-packages '(
+   (vue-mode :location (recipe
+                        :fetcher github
+                        :repo "codefalling/vue-mode"))
+   )
```

Comment:
I'm not sure what you mean by this...
You want me to remove the "encompassing" block or add something else?